modules/rational-ui.el

```diff
@@ -55,18 +55,63 @@ Use a plist with the same key names as accepted by `set-face-attribute'.")
 
 ;;;; Line Numbers
 
+(defcustom rational-ui-line-numbers-enabled-modes
+  '(conf-mode prog-mode)
+  "Modes which should display line numbers."
+  :type 'list
+  :group 'rational)
+
+(defcustom rational-ui-line-numbers-disabled-modes
+  '(org-mode)
+  "Modes which should not display line numbers.
+Modes deriverd from the modes defined in
+`rational-ui-line-number-enabled-modes', but should not display line numbers."
+  :type 'list
+  :group 'rational)
+
+(defun rational-ui--enable-line-numbers-mode ()
+  "Turn on line numbers mode.
+
+Used as hook for modes which should display line numbers."
+  (display-line-numbers-mode 1))
+
+(defun rational-ui--disable-line-numbers-mode ()
+  "Turn off line numbers mode.
+
+Used as hook for modes which should not display line numebrs."
+  (display-line-numbers-mode -1))
+
+(defun rational-ui--update-line-numbers-display ()
+  "Update configuration for line numbers display."
+  (if rational-ui-display-line-numbers
+      (progn
+        (dolist (mode rational-ui-line-numbers-enabled-modes)
+          (message "enabling line numbers for %s" mode)
```

Comment:
Was the message for debugging purposes?  Might be good to remove it for normal use